src/compiler/checker.ts

```diff
@@ -16709,17 +16710,13 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
         return isNoInferType(substitutionType) ? substitutionType.baseType : getIntersectionType([substitutionType.constraint, substitutionType.baseType]);
     }
 
-    function isUnaryTupleTypeNode(node: TypeNode) {
-        return node.kind === SyntaxKind.TupleType && (node as TupleTypeNode).elements.length === 1;
-    }
-
     function getImpliedConstraint(type: Type, checkNode: TypeNode, extendsNode: TypeNode): Type | undefined {
         return isUnaryTupleTypeNode(checkNode) && isUnaryTupleTypeNode(extendsNode) ? getImpliedConstraint(type, (checkNode as TupleTypeNode).elements[0], (extendsNode as TupleTypeNode).elements[0]) :
             getActualTypeVariable(getTypeFromTypeNode(checkNode)) === getActualTypeVariable(type) ? getTypeFromTypeNode(extendsNode) :
             undefined;
     }
 
-    function getConditionalFlowTypeOfType(type: Type, node: Node) {
+    function getFlowTypeOfType(type: Type, node: Node) {

```

Comment:
Why the rename?

Reply:
Originally this was handling flow types in conditional types alone. Later on, this function was expanded with some logic for mapped types but the function's name was never updated. I was on the fence about renaming it but it felt like it's too specific today when perhaps it shouldn't mention conditional types given its expanded scope